test/support/generator/upload-file-in-deal-folder-generator.ts

```diff
@@ -36,10 +33,13 @@ export class UploadFileInDealFolderGenerator extends AbstractGenerator<GenerateV
     };
   }
 
-  protected transformRawValuesToGeneratedValues(valuesList: GenerateValues[]): GenerateResult {
+  protected transformRawValuesToGeneratedValues(valuesList: GenerateValues[], options: GenerateOptions): GenerateResult {
+    const sharepointBaseUrl = options.sharepointBaseUrl ?? ENVIRONMENT_VARIABLES.SHAREPOINT_BASE_URL;
```

Comment:
It does feel slightly odd we haven't included this ability for 
    [ENVIRONMENT_VARIABLES.SHAREPOINT_ESTORE_DOCUMENT_TYPE_ID_FIELD_NAME]: ENVIRONMENT_VARIABLES.SHAREPOINT_ESTORE_DOCUMENT_TYPE_ID_APPLICATION,